src/leo-client-app/src/components/EditSchedules.tsx

```diff
@@ -0,0 +1,185 @@
+
+      //the callback function if calling api endpoint is successful
+    onSuccess: () => {
+      queryClient.invalidateQueries({ queryKey: ["useGetCommandsBySchedule"] });
+      setCurrentSchedule([]);
+    },
+      //the callback function if calling api endpoint is complete
+    onSettled: () => {
+      queryClient.invalidateQueries({ queryKey: ["useGetCommandsBySchedule"] });
+      setCurrentSchedule([]);
+    },
+  });
+
+  // Function will load schedule somewhere, currently console log for POC demo
+  const sendSchedule = async () => {
+    mutate();
+  };
+
+  return (
+    <Box
+      style={{
+        display: "flex",
+        flexDirection: "column",
+        alignItems: "flex-start",
+        gap: "4rem",
+        backgroundColor: "var(--material-theme-sys-dark-background)",
+      }}>
+      <h1 className="material-themedisplaymedium" style={{ width: "100%", color: "var(--material-theme-sys-light-secondary-container", marginLeft: "200px", marginTop: "50px"}}> 
+      Satellite Names </h1>
+     <Box style={{
+          display: "flex",
+          justifyContent: "space-around",
+          alignItems: "flex-start",}}>
+        <Card
+          sx={{
+            minWidth: "200px",
+            border: "2px solid white",
+            borderRadius: "16px",
+            padding: "10px",
+            marginLeft: "250px",
+            backgroundColor: "var(--material-theme-sys-dark-background)"
+          }}>
+          <h2 className="material-themedisplaysmall " style={{ width: "100%", color: "var(--material-theme-sys-light-secondary-container"}}>Valid Commands</h2>
+            {validCommands &&
+              validCommands.length > 0 &&
+              validCommands.map((command, index) => (
+                <Button
+                  key={index}
+                  className="scheduleButton"
+                  onClick={() => addCommand(command)}
+                  style = {{color: "var(--material-theme-sys-light-secondary-container)", fontFamily: "Roboto"}}>
+                  {command}
+                </Button>
+              ))}
+        </Card>
+        <Card
+          sx={{
+            border: "2px solid white",
+            borderRadius: "16px",
+            padding: "10px",
+            overflow: "auto",
+            marginLeft: "220px",
+            backgroundColor: "var(--material-theme-sys-dark-background)",
+          }}>
+          <h2 className="material-themedisplaysmall" style={{ width: "100%", color: "var(--material-theme-sys-light-secondary-container)"}}>Current Schedules</h2>
+          {currentSchedule &&
+            currentSchedule.length > 0 &&
+            currentSchedule.map((command, index) => (
+              <Button
+                key={index}
+                className="removeButton scheduleButton"
+                onClick={() => removeCommand(index)}
+                style = {{color: "var(--material-theme-sys-light-secondary-container)", fontFamily: "Roboto"}}>
```

Comment:
for this one should make a class and toss it in, since it isnt a card or a mui item you can just assign it a css class